src/SearchBar.scss

```diff
@@ -80,6 +82,13 @@
       }
     }
 
+    .ais-SearchBox{
+      width: 100%;
+      width: -moz-available;          /* WebKit-based browsers will ignore this. */
```

Comment:
Unless there is a specific reason for these, I would only keep `width: 100%` here. We already use postcss plugin which would add vendor specific prefixes for the less supported instructions although for width I think we only need 100%.

Reply:
fixed